python/graphstorm/model/node_decoder.py

```diff
@@ -38,20 +38,24 @@ class EntityClassifier(GSLayer):
     norm: str
         Normalization methods. Not used, but reserved for complex node classifier
         implementation. Default: None.
+    bias: bool
+        Whether the node decoder has bias. Default: False.
     """
     def __init__(self,
                  in_dim,
                  num_classes,
                  multilabel,
                  dropout=0,
-                 norm=None):
+                 norm=None,
+                 bias=False):
         super(EntityClassifier, self).__init__()
         self._in_dim = in_dim
         self._num_classes = num_classes
         self._multilabel = multilabel
         self._dropout = dropout
         # TODO(xiangsx): The norm is not used here.
         self._norm = norm
+        self._bias = bias
```

Comment:
```suggestion
        self._use_bias = bias
```
It is misleading if we use _bias as a flag and bias as a nn.Parameter.